docs/source/configuration/logging.mdx

```diff
@@ -66,6 +66,35 @@ JSON-formatted logging provides compatibility with common searchable logging too
 {"timestamp":"2022-03-18T11:46:43.453993Z","level":"INFO","fields":{"message":"Stopped"},"target":"apollo_router"}
 ```
 
+## Basic configuration
+
+### Disclaimer
+
+This is part of an experimental feature, it means any time until it's stabilized (without the prefix `experimental_`) we could change the configuration shape or adding/removing features.
+If you want to give feedback or participate in that feature feel free to join [this discussion on GitHub](https://github.com/apollographql/router/discussions/1961).
+
+By default some of our logs containing sensible data (like request body, response body, headers) are not displayed even if we are in the right log level.
```

Comment:
I'm not familiar with the usage of "sensible" in this context. Do you mean "sensitive"?

Reply:
Oups yes. Sorry :)